src/analytics.js

```diff
@@ -56,3 +58,24 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {!Element} element
+ * @param {!JSONType} config
+ * @param {Document=} parentDoc
+ */
+export function insertAnalyticsElement(element, config, parentDoc) {
+  // TODO(zhouyx): Take extra param to force load analytics extension
```

Comment:
Is this good enough as a TODO at this time? It seems like `amp-ad` requires it, right?